helmchart/sdp/values.yaml

```diff
@@ -4,116 +4,178 @@
+## - Dashboard API: the API used by the front-end UI for all disbursement requests.
+## - Messaging Service: a recurring process that sends text messages to users prompting them to download the wallet selected for a particular disbursement and verify their phone with an OTP
+## - Wallet Registration UI: a web application that collects and verifies the recipient’s OTP code and verification information via Stellar’s SEP-24: Hosted Deposit and Withdrawal protocol
+## @descriptionEnd
 sdp:
-  # =========================== START sdp.route ===========================
+  ## @extra sdp.route Configuration related to the routing of the SDP service.
+  ## @param sdp.route.schema Protocol scheme used for the service. Can be "http" or "https".
+  ## @param sdp.route.domain Public domain/address of the SDP service. If using localhost, consider including the port as part of the domain.
+  ## @param sdp.route.port Primary port on which the SDP service listens.
+  ## @param sdp.route.metricsPort Port dedicated to metrics collection for the SDP service.
   route:
     schema: "https"
-    # The domain is the public address of that service. For localhost you will want to include the port as part of the domain.
-    # domain: "localhost:8000"
     domain: sdp.localhost.com
     port: "8000"
     metricsPort: "8002"
-  # =========================== START sdp.image ===========================
+
+  ## @extra sdp.image Configuration related to the Docker image used by the SDP service.
+  ## @param sdp.image.repository Docker image repository for the SDP backend service.
+  ## @param sdp.image.pullPolicy Image pull policy for the SDP service. For locally built images, consider using "Never" or "IfNotPresent".
+  ## @param sdp.image.tag Docker image tag for the SDP service. If set, this overrides the default value from `.Chart.AppVersion`.
   image:
-    # replace the `repository` with the actual image name.
     repository: stellar/stellar-disbursement-platform-backend
-    # for locally built images, use `pullPolicy: Never or IfNotPresent`.
     pullPolicy: IfNotPresent
-    # If `tag` is set, it'll override the default value set in `.Chart.AppVersion`.
     tag: "latest"
 
-  # =========================== START sdp.deployment ===========================
+  ## @extra sdp.deployment Configuration related to the deployment of the SDP service.
+  ## @param sdp.deployment.annotations Annotations to be added to the deployment.
+  ## @extra sdp.deployment.podAnnotations Annotations specific to the pods, mainly for integration with monitoring tools like Prometheus.
+  ## @param sdp.deployment.podAnnotations.prometheus.io/path The path where Prometheus metrics are exposed.
+  ## @param sdp.deployment.podAnnotations.prometheus.io/port The port on which Prometheus metrics are exposed. This should match the SDP metrics port.
+  ## @param sdp.deployment.podAnnotations.prometheus.io/scrape Indicates if Prometheus should scrape metrics from this pod.
```

Comment:
The podAnnotations are applied as is, so we shouldn't mention things like `prometheus.io/path` here. Here is how that's being used:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/2945cef15ca51f0fbba99b3c45cbdbb4845cd329/helmchart/sdp/templates/02-deployment-sdp.yaml#L27-L30

The same applies for all:
- [ ] sdp.deployment.podAnnotations
- [ ] anchorPlatform.deployment.podAnnotations
- [ ] tss.deployment.podAnnotations